.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node: [12.x, 14.x]
+      fail-fast: false
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set node version to ${{ matrix.node }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Versions
+        run: yarn versions
+
+      - name: Install dependencies
+        run: yarn install #--frozen-lockfile
+
+      # - name: Check
+      #   run: yarn check
+
+      # - name: Lint
+      #   run: yarn lint
+
+      - name: Audit dependencies
+        run: yarn audit --groups dependencies
+
+      # - name: Audit devDependencies
+      #   run: yarn audit --groups devDependencies
+
+      # - name: Audit peerDependencies
+      #   run: yarn audit --groups peerDependencies
+
+      # - name: Audit optionalDependencies
+      #   run: yarn audit --groups optionalDependencies
+
+      - name: Build
+        run: yarn build
+
+      - name: Test
+        run: yarn test
+
+      - name: Codecov
+        run: yarn codecov
+
+      - name: Codeclimate
+        run: yarn codeclimate-test-reporter < ./coverage/lcov.info
```

Comment:
Codeclimate requests `CODECLIMATE_REPO_TOKEN`

Reply:
I'll give you the token in private and let's check if you have access to code climate as they use GitHub backed auth.

Reply:
mh 🤔 https://docs.codeclimate.com/docs/command-line-interface#error-no-codeclimate_repo_token-found-a-codeclimate_repo_token-must-be-specified-as-an-environment-variable

Do we already need it? Or is the dependency outdated?

Reply:
I would suggest to update and bring the project to the newest state in a dedicated PR

There I will update all package deps and try to fix all error's to bring the current state of the project to a working state

_This could also help me onboarding into what the current idea of the project is/was_

Reply:
Sounds like a plan.